content/fundamentals/unit-testing.md

```diff
@@ -198,6 +198,8 @@ describe('CatsController', () => {
 
 You can also retrieve these mocks out of the testing container as you normally would custom providers, `moduleRef.get(CatsService)`.
 
+Do note that `REQUEST` and `INQUIRER` providers cannot be auto-mocked because they're already defined in the context, but they can be _overwritten_ with a custom provider or a `.overrideProvider`.
```

Comment:
```suggestion
> info **Hint**`REQUEST` and `INQUIRER` providers cannot be auto-mocked because they're already pre-defined in the context. However, they can be _overwritten_ using the custom provider syntax or by utilizing the `.overrideProvider` method.
```